ledger/participant-state-kv-errors/src/main/scala/com/daml/ledger/error/definitions/kv/KvErrors.scala

```diff
@@ -90,6 +110,24 @@ object KvErrors extends ErrorGroup()(ErrorGroups.rootErrorClass) {
       }
     }
 
+    @Explanation(
+      "The record time is not within bounds for reasons such as " +
+        "excessive latency, excessive clock skew between the participant and the committer " +
+        "or a time model that is too restrictive."
+    )
+    @Resolution("Retry the submission or contact the participant operator.")
+    object RecordTimeOutOfBounds
+        extends ErrorCode(
+          id = "RECORD_TIME_OUT_OF_BOUNDS",
+          ErrorCategory.ContentionOnSharedResources, // It may succeed at a later time
+        ) {
+      case class Reject(
+          override val definiteAnswer: Boolean,
```

Comment:
Why do we need to override this field?

Reply:
Good point, I had the wrong impression that it's an abstract. Will open a short PR to remove it

Reply:
https://github.com/digital-asset/daml/pull/13728